resources/views/admin/partials/featured-related.blade.php

```diff
@@ -36,4 +36,30 @@
             ],
         ],
     ])
+
+    @php
+        $autoRelated = collect($item->related($item->id))->unique('id')->filter();
+
+        $featuredRelated = collect($item->getFeaturedRelated())->pluck('item');
+        $featuredRelatedIds = $featuredRelated->pluck('id');
+
+        // Remove featured related items from auto related items
+        if ($featuredRelatedIds->isNotEmpty()) {
+            $autoRelated = $autoRelated->reject(function ($relatedItem) use ($featuredRelatedIds) {
+                return ($relatedItem !== null && ($featuredRelatedIds->contains($relatedItem->id) || $featuredRelatedIds->contains($relatedItem->datahub_id)));
+            });
+        }
+    @endphp
```

Comment:
Move this into the `formData` methods of the Admin Controllers in which this is applicable and pass in `$autoRelated`. Then on the next line below check `isset($autoRelated) && count($autoRelated) > 0`.

In the MVC model of coding, logic like this shouldn't live in views.